Sources/Support/PaywallExtensions.swift

```diff
@@ -124,8 +124,16 @@ private extension View {
                     Logger.appleWarning(Strings.configure.sk2_required_for_swiftui_paywalls)
                 }
 
+                // Find offering context from a matching package
+                let offeringContext = offering.availablePackages.first {
+                    $0.storeProduct.productIdentifier == product.id
```

Comment:
Hmm this could potentially be wrong if the offering has multiple packages with the same product. It would likely be a misconfiguration so maybe it's ok to assume this in almost all cases though...

Reply:
This is true! However, both packages would actually have the same `PresentedOfferingContext` in this situation so this is "technically" okay 🤷‍♂️  I'll make a note of that in the comments. But this is also only used by the StoreKit Views so it probably doesn't get used too often.